oak/server/rust/oak_runtime/src/runtime/mod.rs

```diff
@@ -452,9 +452,18 @@ impl Runtime {
             }
             let statuses = self.readers_statuses(node_id, &readers);
 
+            let any_invalid = statuses.iter().any(|&s| {
```

Comment:
@tiziano88 Is this what you are looking for in #803?
Also, by invalid do you mean a channel handle that has status `InvalidChannel`? or does `Orphaned` count as invalid too?

Reply:
Perhaps also return immediately if any channel is Orphaned? I haven't thought enough about it though, happy to go with your suggestion.

Reply:
I agree. That is what I am doing now (not pushed yet... an abitest is failing).

Reply:
Sorry to come to this late, but I don't think this (i.e. #803) is a good idea.  Just returning an overall `Err` does not give the calling application enough information to work with; for example, if an app has 100 channels that it's waiting on, and one of them happens to be closed (due to some bug or other), then it's really difficult to figure out *which one*.

Returning an immediate `Err` is any one of the 100 channels is orphaned is even worse; now the Node is left with the prospect of coding up a binary chop to figure out which channels have been closed by other Nodes while it wasn't looking.